de.metas.payment.esr/pom.xml

```diff
@@ -111,38 +111,20 @@
 						</configuration>
 					</execution>
 					
-					<!-- NOTE about "<clearOutputDir>false</clearOutputDir>" further down: don't clear the directory because else we will lose previously generated files	-->
-					<!--
-						https://github.com/metasfresh/metasfresh/issues/1903
-						We don't need jaxb code for these two XSD versions. 
-						They are so similar (at least the parts we use) that we rather trick jaxb into thinking that XML created according to ..001.04 and ..001.05 i actually ..001.06 as well.
 					<execution>
```

Comment:
with this change, the build should now pick up the new XSD file, but the package names differ from the ones in the branch you are porting from. All in all, please make sure it compiles in your workspace :-)